conf/distro/include/emlinux.inc

```diff
@@ -27,6 +27,8 @@ DISTRO_EXTRA_RDEPENDS += " ${EMLINUX_DEFAULT_EXTRA_RDEPENDS}"
 LINUX_GIT_URI ?= "git://git.kernel.org/pub/scm/linux/kernel/git/cip"
```

Comment:
As I see you added a space for separating variables,
what about adding a comment here specifying that this space if for kernel related commons variable